build.sh

```diff
@@ -3,38 +3,15 @@
 set -euxo pipefail
 
 output_dir=$1
-
-git_commit_hash=${2:-$(git rev-parse HEAD)}
-
-git_version_string=${3:-$(git log --date=format:%Y%m%d --pretty=~git%cd.%h -n 1)}
-
+git_commit_hash=${2:-$(git rev-parse --short HEAD)}
 build_number=${GITHUB_RUN_NUMBER:=0}
 
-ros_distro=${ROS_DISTRO:=foxy}
-
-iname=${PACKAGE_NAME:=px4-msgs}
-
-iversion=${PACKAGE_VERSION:=latest}
-
 docker build \
-  --build-arg UID=$(id -u) \
-  --build-arg GID=$(id -g) \
-  --build-arg ROS_DISTRO=${ros_distro} \
-  --build-arg PACKAGE_NAME=${iname} \
-  --pull \
-  -f Dockerfile.build_env -t "${iname}_build:${iversion}" .
-
-docker run \
-  --rm \
-  -v $(pwd):/${iname}/sources \
-  ${iname}_build:${iversion} \
-  ./packaging/package.sh \
-  -b ${build_number} \
-  -g ${git_commit_hash} \
-  -v ${git_version_string}
+    --build-arg BUILD_NUMBER=${build_number} \
+    -t "ghcr.io/tiiuae/tii-px4-msgs:sha-${git_commit_hash}" .
```

Comment:
Minor stylistic nag: since we don't actually push any Docker images, semantically it might be a bit misleading to tag the resulting image with a valid-looking container image refernce to GHCR in a package that doesn't exist.

The tag argument is not required, but I see it is uses as a reference in code below. In such case a "dummy" reference like "px4-msgs" could be better which should indicate that it is not a publicly retrievable image.

This might be subjective feedback, so feel free to ignore :)

Reply:
Good comment!! I will change the name of the image to not be confusing. :)

Reply:
Done.